src/domains/shared-app/04_apim_reporting_orgs_enrollment.tf

```diff
@@ -0,0 +1,75 @@
+###########################################
+## Product POC REPORTING ORGS ENROLLMENT ##
+###########################################
+
+locals {
+  apim_poc_service_api = {
+    display_name          = "Reporting orgs enrollment - POC pagoPA"
+    description           = "API to support POC"
+    path                  = "shared/poc-reporting-orgs-enrollment"
+    subscription_required = true
+    service_url           = null
+  }
+}
+
+##############
+## Products ##
+##############
+
+module "apim_poc_product" {
```

Comment:
I think we should have poc only in dev...so we should use `count` to apply them only in that env